homeassistant/components/geo_json_events/const.py

```diff
@@ -1,14 +1,19 @@
 """Define constants for the GeoJSON events integration."""
 from __future__ import annotations
 
-from datetime import timedelta
 from typing import Final
 
+from homeassistant.const import Platform
+
 DOMAIN: Final = "geo_json_events"
 
+PLATFORMS: Final = [Platform.GEO_LOCATION]
+
 ATTR_EXTERNAL_ID: Final = "external_id"
 DEFAULT_RADIUS_IN_KM: Final = 20.0
-DEFAULT_SCAN_INTERVAL: Final = timedelta(minutes=5)
+DEFAULT_RADIUS_IN_M: Final = 20000.0
+DEFAULT_SCAN_INTERVAL: Final = 300
+FEED: Final = "feed"
```

Comment:
```suggestion
```

Reply:
OK, removing the use of "feed" everywhere.